src/internal/connector/graph_connector_helper_test.go

```diff
@@ -814,20 +816,23 @@ func makeOneDriveBackupSel(
 	dests []destAndCats,
 ) selectors.Selector {
 	toInclude := [][]selectors.OneDriveScope{}
-	resourceOwners := []string{}
+	resourceOwners := map[string]struct{}{}
 
 	for _, d := range dests {
+		resourceOwners[d.resourceOwner] = struct{}{}
+
+		// nil owners here, we'll need to stitch this together
+		// below after the loops are complete.
 		sel := selectors.NewOneDriveBackup(nil)
```

Comment:
if we don't remove the users parameter from scope creation (i.e. `sel.Folders`) then it seems like it would just be better to change what the `toInclude` map holds so we don't have to play so many tricks here. This is test code so it doesn't need to be super efficient in all ways